### ISSUE TITLE
What Guthub tag numbering scheme should we use in the future?

### ISSUE THREAD

**Issue by MarkRivers** [epics-motor/motorSymetrie#7]
The RELEASE.md files lists the following as the existing release numbers:

- 5.0.210830
- 5.0.201028
- 5.0.191103

Do the final 6 digits in these encode the release date?

This is currently just one Github tag, R1-0 which I applied to the version I received from Symetrie.

The question is what numbering scheme should we use for Github tags going forward?  Should we use the prior convention of 5.0.date or R1-1, R1-2, which is more conventional for EPICS modules?

**Comment by prjemian** [comment]
Semantic Versioning  (https://semver.org) is one approach.

**Comment by JohannLamaury** [comment]
> Do the final 6 digits in these encode the release date?

That's correct. Our versioning semantic for releases is composed of MAJOR.MINOR.DATE
where DATE is defined as YYMMDD
